### DISCUSSION TITLE
A regression in JReleaser?

### DISCUSSION THREAD

**Discussion by triceo** [jreleaser/jreleaser#1467] [Q&A]
I use JReleaser 1.7.0, yet I'm still seeing [this issue](https://github.com/jreleaser/jreleaser/issues/1397).

    [ERROR]     [artifactory] PomChecker detected errors <redacted>.pom
    [ERROR] <description> is not defined in POM. Will use value from parent: 

(The `verifyPom: false` workaround still works.)

Unfortunately I can not share the entire GHA because it is in a private repo, but I am attaching the (redacted) JReleaser trace log at least.

Have I misconfigured JReleaser, or was the above never really fixed?
[trace.log](https://github.com/jreleaser/jreleaser/files/12355981/trace.log)

**Comment by aalmiray** [comment]
It might be pomchecker itself. You can switch it to an older version.
